apps/prairielearn/assets/scripts/navbarClient.ts

```diff
@@ -0,0 +1,154 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+import Cookies from 'js-cookie';
+
+const expires = 30; // in units of days
+
+onDocumentReady(() => {
+  const usernameNav = document.getElementById('username-nav');
+  const accessAsAdministrator = usernameNav.dataset.accessAsAdministrator === 'true';
+  const viewType = usernameNav.dataset.viewType;
+  const authnCourseRole = usernameNav.dataset.authnCourseRole;
+  const authnCourseInstanceRole = usernameNav.dataset.authnCourseInstanceRole;
+  // Note: this corresponds to `user_with_requested_uid_has_instructor_access_to_course_instance`.
+  // Be careful what you use it for.
+  const hasInstructorAccess = usernameNav.dataset.hasInstructorAccess === 'true';
+
+  document.querySelector('#navbar-load-from-disk')?.addEventListener('click', () => {
+    Cookies.remove('pl_requested_uid', { path: '/' });
```

Comment:
Is there a particular reason we're always specifying the path as `/`? AFAICT this is the default behavior and everything behaves the same without it.

Reply:
One option to avoid this is to [use default attributes](https://www.npmjs.com/package/js-cookie#setting-up-defaults), which would also remove the requirement to have the expires constant.

Reply:
That's a good suggestion, especially once we start explicitly specifying the domain. I'm going to merge this as-is and will consider doing this in the followup PR.